tests/unit/yaml/test-index.yml

```diff
@@ -3,7 +3,7 @@ with:
   index_filename: tmp2
 metas:
   name: test2
-  workspace: $JINA_TEST_INDEX
+  workspace: $TEST_WORKDIR
```

Comment:
We arr trying to set the habit to prefix env variablws with JINA so please keep it as it was

Reply:
I'll change it back